profiles/shivamp1998.json

```
@@ -0,0 +1,6 @@
+{
+    "image" : "https://user-images.githubusercontent.com/48366063/193478291-efc11969-6ba4-41bb-bfdf-89d1644afdba.png",
+    "issueId" : 1454,
+    "name" : "sarvesh pandey",
+    "username" : "shivamp1998"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

added newline